tox.ini

```diff
@@ -1,10 +1,11 @@
 [tox]
 envlist =
-    py27,py34,py35,py36,py37
+    py27,py34,py35,py36,py37,py38
 
 [testenv]
 deps =
     zope.testrunner
+    zope.interface>=5.0.0
```

Comment:
IMHO if zope.interface >= 5.0.0 is a requirement for tests, it should be part of the [test] extra in setup.py.

Reply:
That was again for a failed attempt for 3.4. I have removed it.